GoRogue.UnitTests/MapGeneration/PolygonAreaTests.cs

```diff
@@ -58,6 +88,40 @@ public struct PolygonTestCase
                 (11,4),(10,7), (11,9),(16,9),(15,11),(15,13),(1,13))
         };
 
+        private string GetPolygonString(PolygonArea region)
+        {
+            var bounds = region.Bounds;
+            var final = new StringBuilder();
+
+            // Generate x scale
+            final.Append(' ');
+            for (int i = bounds.MinExtentX; i < bounds.MaxExtentX; i++)
+                final.Append($" {i + bounds.MinExtentX}");
+            final.Append('\n');
+
+            for (int y = bounds.MinExtentY; y <= bounds.MaxExtentY; y++)
+            {
+                final.Append($"{y + bounds.MinExtentY}");
+                for (int x = bounds.MinExtentX; x <= bounds.MaxExtentX; x++)
+                {
+                    if (region.Corners.Contains((x, y)))
```

Comment:
Note: since one of the caveats is that the unit tests are very slow, it might be possible to get some more speed if we do one of the following:

1. Create a HashSet at the top of this function that contains everything in `region.Corners`; that way contains does't take so long

OR

2. Move this if-check to the bottom of the if-else check, since inner/outer points will catch most of the points so it will avoid executing this expensive one for each location